src/SwaggerProvider.Runtime/ProvidedApiClientBase.fs

```diff
@@ -2,39 +2,42 @@ namespace Swagger
 
 open System
 open System.Net.Http
-open Newtonsoft.Json
-
-open Swagger.Serialization
+open System.Text.Json
+open System.Text.Json.Serialization
 
 type OpenApiException(code:int, description:string) =
     inherit Exception(description)
     member __.StatusCode = code
     member __.Description = description
 
-type ProvidedApiClientBase(httpClient: HttpClient) =
-    let jsonSerializerSettings =
-        let settings =
-            JsonSerializerSettings(
-                NullValueHandling = NullValueHandling.Ignore,
-                Formatting = Formatting.None)
+type ProvidedApiClientBase(httpClient: HttpClient, options: JsonSerializerOptions) =
+
 #if TP_RUNTIME
-        [
-            OptionConverter () :> JsonConverter
-            ByteArrayConverter () :> JsonConverter
-        ]
-        |> List.iter settings.Converters.Add
+    let options =
+        if isNull options then
+            let options = JsonSerializerOptions()
+            [
+                JsonFSharpConverter(
```

Comment:
Why here used not default converters config `JsonFSharpConverter()`?

Reply:
It only exists in .NET 5 as far as I know and it is not as powerful as `FSharp.SystemtextJson`

Reply:
I mean, why do you use
```fsharp
            let options = JsonSerializerOptions()
            [
                JsonFSharpConverter(
                    JsonUnionEncoding.InternalTag
                    ||| JsonUnionEncoding.NamedFields
                    ||| JsonUnionEncoding.UnwrapSingleCaseUnions
                    ||| JsonUnionEncoding.UnwrapRecordCases
                    ||| JsonUnionEncoding.UnwrapOption) :> JsonConverter
            ]
            |> List.iter options.Converters.Add
```

as a default configuration, instead of 

```fsharp
            let options = JsonSerializerOptions()
            [
                JsonFSharpConverter() :> JsonConverter
            ]
            |> List.iter options.Converters.Add
```

for some reason, author of `FSharp.SystemTextJson` choose different default configuration for `JsonFSharpConverter `

Reply:
We can go with the default configuration.
I don't know why he did that.
I see his defaults too verbose in JSON